services/api/src/typeDefs.js

```diff
@@ -1006,6 +1025,43 @@ const typeDefs = gql`
     parentGroup: GroupInput
   }
 
+
+
+  input BillingModifierInput {
```

Comment:
Naming convention here would be AddBillingModifierInput

Reply:
addressed in: 18a1d2012c92ca915b841bc0ef6df11874840484

Reply:
Addressed